# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2001-2023, Python Software Foundation
# This file is distributed under the same license as the Python package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
# Translators:
# 8af080f2e6702c64bedd01873aed27e8_25aec74 <fd99d14239138a1f987c22e7736dde81_777475>, 2019
# Sean Chao <seanchao0804@gmail.com>, 2020
# helloworldSB <warningfor163@foxmail.com>, 2021
# Dai Xu <daixu61@hotmail.com>, 2022
# Freesand Leo <yuqinju@163.com>, 2022
# Bryan不可思议, 2023
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Python 3.7\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-06-16 16:41+0000\n"
"PO-Revision-Date: 2019-09-01 03:38+0000\n"
"Last-Translator: Bryan不可思议, 2023\n"
"Language-Team: Chinese (China) (https://app.transifex.com/python-doc/"
"teams/5390/zh_CN/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: zh_CN\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:6
msgid "Integer Objects"
msgstr "整数型对象"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:11
msgid ""
"All integers are implemented as \"long\" integer objects of arbitrary size."
msgstr "所有整数都使用以任意大小的长整数对象表示。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:13
msgid ""
"On error, most ``PyLong_As*`` APIs return ``(return type)-1`` which cannot "
"be distinguished from a number.  Use :c:func:`PyErr_Occurred` to "
"disambiguate."
msgstr ""
"在出错时，大多数 ``PyLong_As*`` API返回 ``(返回值类型)-1`` ，无法与一般的数字"
"区分开来。请使用 :c:func:`PyErr_Occurred` 来区分。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:18
msgid "This subtype of :c:type:`PyObject` represents a Python integer object."
msgstr "表示 Python 整数对象的 :c:type:`PyObject` 子类型。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:23
msgid ""
"This instance of :c:type:`PyTypeObject` represents the Python integer type. "
"This is the same object as :class:`int` in the Python layer."
msgstr ""
"这个 :c:type:`PyTypeObject` 的实例表示 Python 的整数类型。与Python层中的 :"
"class:`int` 相同。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:29
msgid ""
"Return true if its argument is a :c:type:`PyLongObject` or a subtype of :c:"
"type:`PyLongObject`."
msgstr ""
"如果参数是 :c:type:`PyLongObject` 或 :c:type:`PyLongObject` 的子类型，返回 "
"true 。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:35
msgid ""
"Return true if its argument is a :c:type:`PyLongObject`, but not a subtype "
"of :c:type:`PyLongObject`."
msgstr ""
"如果参数属于 :c:type:`PyLongObject`，但不是 :c:type:`PyLongObject` 的子类型则"
"返回真值。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:41
msgid ""
"Return a new :c:type:`PyLongObject` object from *v*, or ``NULL`` on failure."
msgstr ""
"由 *v* 返回一个新的 :c:type:`PyLongObject` 对象，失败时返回 ``NULL`` 。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:43
msgid ""
"The current implementation keeps an array of integer objects for all "
"integers between ``-5`` and ``256``, when you create an int in that range "
"you actually just get back a reference to the existing object. So it should "
"be possible to change the value of ``1``.  I suspect the behaviour of Python "
"in this case is undefined. :-)"
msgstr ""

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:52
msgid ""
"Return a new :c:type:`PyLongObject` object from a C :c:type:`unsigned long`, "
"or ``NULL`` on failure."
msgstr ""
"由 C :c:type:`unsigned long` 类型返回一个新的 :c:type:`PyLongObject` 对象，失"
"败时返回 ``NULL`` 。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:58
msgid ""
"Return a new :c:type:`PyLongObject` object from a C :c:type:`Py_ssize_t`, or "
"``NULL`` on failure."
msgstr ""
"从 C :c:type:`Py_ssize_t` 类型返回一个新的 :c:type:`PyLongObject` 对象，如果"
"失败则返回``NULL``。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:64
msgid ""
"Return a new :c:type:`PyLongObject` object from a C :c:type:`size_t`, or "
"``NULL`` on failure."
msgstr ""
"从 C :c:type:`size_t` 返回一个新的 :c:type:`PyLongObject` 对象，如果失败则返"
"回``NULL``。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:70
msgid ""
"Return a new :c:type:`PyLongObject` object from a C :c:type:`long long`, or "
"``NULL`` on failure."
msgstr ""
"从 C :c:type:`long long` 返回一个新的 :c:type:`PyLongObject` 对象，失败时返回"
"``NULL``。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:76
msgid ""
"Return a new :c:type:`PyLongObject` object from a C :c:type:`unsigned long "
"long`, or ``NULL`` on failure."
msgstr ""
"从 C :c:type:`unsigned long long` 返回一个新的 :c:type:`PyLongObject` 对象，"
"失败时返回``NULL``。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:82
msgid ""
"Return a new :c:type:`PyLongObject` object from the integer part of *v*, or "
"``NULL`` on failure."
msgstr ""
"从 *v* 的整数部分返回一个新的 :c:type:`PyLongObject` 对象，如果失败则返回"
"``NULL``。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:88
msgid ""
"Return a new :c:type:`PyLongObject` based on the string value in *str*, "
"which is interpreted according to the radix in *base*.  If *pend* is non-"
"``NULL``, *\\*pend* will point to the first character in *str* which follows "
"the representation of the number.  If *base* is ``0``, *str* is interpreted "
"using the :ref:`integers` definition; in this case, leading zeros in a non-"
"zero decimal number raises a :exc:`ValueError`. If *base* is not ``0``, it "
"must be between ``2`` and ``36``, inclusive.  Leading spaces and single "
"underscores after a base specifier and between digits are ignored.  If there "
"are no digits, :exc:`ValueError` will be raised."
msgstr ""
"根据 *str* 字符串值返回一个新的 :c:type:`PyLongObject` ，*base* 指定基数。如"
"果 *pend* 不是 ``NULL`` ， */*pend* 将指向 *str* 中表示这个数字部分的后面的第"
"一个字符。如果 *base* 是 ``0`` ， *str* 将使用 :ref:`integers` 定义来解释；在"
"这种情况下，一个非零的十进制数中的前导零会引发一个 :exc:`ValueError` 。如果 "
"*base* 不是 ``0`` ，它必须在 ``2`` 和` `36`` 之间，包括2和36。基数说明符后以"
"及数字之间的前导空格、单下划线将被忽略。如果没有数字，将引发 :exc:"
"`ValueError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:101
msgid ""
"Convert a sequence of Unicode digits to a Python integer value.  The Unicode "
"string is first encoded to a byte string using :c:func:"
"`PyUnicode_EncodeDecimal` and then converted using :c:func:"
"`PyLong_FromString`."
msgstr ""

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:107
msgid ""
"Part of the old-style :c:type:`Py_UNICODE` API; please migrate to using :c:"
"func:`PyLong_FromUnicodeObject`."
msgstr ""
"旧的 :c:type:`Py_UNICODE` API的一部分;请迁移到使用 :c:func:"
"`PyLong_FromUnicodeObject` 。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:112
msgid ""
"Convert a sequence of Unicode digits in the string *u* to a Python integer "
"value.  The Unicode string is first encoded to a byte string using :c:func:"
"`PyUnicode_EncodeDecimal` and then converted using :c:func:"
"`PyLong_FromString`."
msgstr ""

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:122
msgid ""
"Create a Python integer from the pointer *p*. The pointer value can be "
"retrieved from the resulting value using :c:func:`PyLong_AsVoidPtr`."
msgstr ""
"从指针 *p* 创建一个 Python 整数。可以使用 :c:func:`PyLong_AsVoidPtr` 返回的指"
"针值。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:133
#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:145
msgid ""
"Return a C :c:type:`long` representation of *obj*.  If *obj* is not an "
"instance of :c:type:`PyLongObject`, first call its :meth:`__int__` method "
"(if present) to convert it to a :c:type:`PyLongObject`."
msgstr ""
"返回 *obj* 的 C :c:type:`long` 表示形式。 如果 *obj* 不是 :c:type:"
"`PyLongObject` 的实例，则会先调用其 :meth:`__int__` 方法（如果有的话）将其转"
"换为 :c:type:`PyLongObject`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:137
msgid ""
"Raise :exc:`OverflowError` if the value of *obj* is out of range for a :c:"
"type:`long`."
msgstr ""
"如果 *obj* 的值溢出了 :c:type:`long` 的范围，会抛出 :exc:`OverflowError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:140
#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:154
#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:169
#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:183
#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:200
msgid "Returns ``-1`` on error.  Use :c:func:`PyErr_Occurred` to disambiguate."
msgstr "发生错误时返回 ``-1`` 。使用 :c:func:`PyErr_Occurred` 来消歧义。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:149
msgid ""
"If the value of *obj* is greater than :const:`LONG_MAX` or less than :const:"
"`LONG_MIN`, set *\\*overflow* to ``1`` or ``-1``, respectively, and return "
"``-1``; otherwise, set *\\*overflow* to ``0``.  If any other exception "
"occurs set *\\*overflow* to ``0`` and return ``-1`` as usual."
msgstr ""
"如果 *obj* 的值大于 :const:`LONG_MAX` 或小于 :const:`LONG_MIN`，则会把 "
"*\\*overflow* 分别置为``1`` 或 ``-1``，并返回 ``1``；否则，将 *\\*overflow*  "
"置为 `0`。如果发生其他异常，则会按常规把 *\\*overflow* 置为 ``0``，并返回 "
"``-1``。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:162
#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:174
msgid ""
"Return a C :c:type:`long long` representation of *obj*.  If *obj* is not an "
"instance of :c:type:`PyLongObject`, first call its :meth:`__int__` method "
"(if present) to convert it to a :c:type:`PyLongObject`."
msgstr ""
"返回 *obj* 的 C :c:type:`long long` 表示形式。 如果 *obj* 不是 :c:type:"
"`PyLongObject` 的实例，则会先调用其 :meth:`__int__` 方法（如果有的话）将其转"
"换为 :c:type:`PyLongObject`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:166
msgid ""
"Raise :exc:`OverflowError` if the value of *obj* is out of range for a :c:"
"type:`long long`."
msgstr "如果 *obj* 值超出 :c:type:`long long` ，触发 :exc:`OverflowError`"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:178
msgid ""
"If the value of *obj* is greater than :const:`PY_LLONG_MAX` or less than :"
"const:`PY_LLONG_MIN`, set *\\*overflow* to ``1`` or ``-1``, respectively, "
"and return ``-1``; otherwise, set *\\*overflow* to ``0``.  If any other "
"exception occurs set *\\*overflow* to ``0`` and return ``-1`` as usual."
msgstr ""

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:194
msgid ""
"Return a C :c:type:`Py_ssize_t` representation of *pylong*.  *pylong* must "
"be an instance of :c:type:`PyLongObject`."
msgstr ""
"返回 *pylong* 的 C 语言 :c:type:`Py_ssize_t` 形式。*pylong* 必须是 :c:type:"
"`PyLongObject` 的实例。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:197
msgid ""
"Raise :exc:`OverflowError` if the value of *pylong* is out of range for a :c:"
"type:`Py_ssize_t`."
msgstr ""
"如果 *pylong* 的值超出了 :c:type:`Py_ssize_t` 的取值范围则会引发 :exc:"
"`OverflowError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:209
msgid ""
"Return a C :c:type:`unsigned long` representation of *pylong*.  *pylong* "
"must be an instance of :c:type:`PyLongObject`."
msgstr ""
"返回 *pylong* 的 C :c:type:`unsigned long` 形式。 *pylong* 必须是 :c:type:"
"`PyLongObject` 的实例。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:212
msgid ""
"Raise :exc:`OverflowError` if the value of *pylong* is out of range for a :c:"
"type:`unsigned long`."
msgstr ""
"如果 *pylong* 的值超出了 :c:type:`unsigned long` 的取值范围，则会触发 :exc:"
"`OverflowError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:215
msgid ""
"Returns ``(unsigned long)-1`` on error. Use :c:func:`PyErr_Occurred` to "
"disambiguate."
msgstr ""
"出错时返回 ``(unsigned long)-1`` ，请利用 :c:func:`PyErr_Occurred` 获取详细信"
"息。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:225
msgid ""
"Return a C :c:type:`size_t` representation of *pylong*.  *pylong* must be an "
"instance of :c:type:`PyLongObject`."
msgstr ""
"返回 *pylong* 的 C 语言 :c:type:`size_t` 形式。*pylong* 必须是 :c:type:"
"`PyLongObject` 的实例。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:228
msgid ""
"Raise :exc:`OverflowError` if the value of *pylong* is out of range for a :c:"
"type:`size_t`."
msgstr ""
"如果 *pylong* 的值超出了 :c:type:`size_t` 的取值范围则会引发 :exc:"
"`OverflowError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:231
msgid ""
"Returns ``(size_t)-1`` on error. Use :c:func:`PyErr_Occurred` to "
"disambiguate."
msgstr ""
"出错时返回  ``(size_t)-1`` ，请利用 :c:func:`PyErr_Occurred` 获取详细信息。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:240
msgid ""
"Return a C :c:type:`unsigned long long` representation of *pylong*.  "
"*pylong* must be an instance of :c:type:`PyLongObject`."
msgstr ""
"返回 *pylong* 的 C 语言 :c:type:`unsigned long long`  形式。*pylong* 必须是 :"
"c:type:`PyLongObject` 的实例。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:243
msgid ""
"Raise :exc:`OverflowError` if the value of *pylong* is out of range for an :"
"c:type:`unsigned long long`."
msgstr ""
"如果 *pylong* 的值超出了 :c:type:`unsigned long long` 的取值范围则会引发 :"
"exc:`OverflowError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:246
msgid ""
"Returns ``(unsigned long long)-1`` on error. Use :c:func:`PyErr_Occurred` to "
"disambiguate."
msgstr ""
"出错时返回 ``(unsigned long long)-1``，请利用 :c:func:`PyErr_Occurred` 获取详"
"细信息。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:249
msgid ""
"A negative *pylong* now raises :exc:`OverflowError`, not :exc:`TypeError`."
msgstr ""
"现在 *pylong* 为负值会触发  :exc:`OverflowError`，而不是 :exc:`TypeError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:255
msgid ""
"Return a C :c:type:`unsigned long` representation of *obj*.  If *obj* is not "
"an instance of :c:type:`PyLongObject`, first call its :meth:`__int__` method "
"(if present) to convert it to a :c:type:`PyLongObject`."
msgstr ""
"返回 *obj* 的 C :c:type:`unsigned long` 表示形式。 如果 *obj* 不是 :c:type:"
"`PyLongObject` 的实例，则会先调用其 :meth:`__int__` 方法（如果有的话）将其转"
"为 :c:type:`PyLongObject`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:259
msgid ""
"If the value of *obj* is out of range for an :c:type:`unsigned long`, return "
"the reduction of that value modulo ``ULONG_MAX + 1``."
msgstr ""
"如果 *obj* 的值超出了 :c:type:`unsigned long` 的范围，则返回该值对 "
"``ULONG_MAX + 1`` 求模的差值。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:262
msgid ""
"Returns ``(unsigned long)-1`` on error.  Use :c:func:`PyErr_Occurred` to "
"disambiguate."
msgstr ""
"出错时返回 ``(unsigned long)-1``，请利用 :c:func:`PyErr_Occurred` 辨别具体问"
"题。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:268
msgid ""
"Return a C :c:type:`unsigned long long` representation of *obj*.  If *obj* "
"is not an instance of :c:type:`PyLongObject`, first call its :meth:`__int__` "
"method (if present) to convert it to a :c:type:`PyLongObject`."
msgstr ""
"返回 *obj* 的 C :c:type:`unsigned long long` 表示形式。 如果 *obj* 不是 :c:"
"type:`PyLongObject` 的实例，则会先调用其 :meth:`__int__` 方法（如果有的话）将"
"其转为 :c:type:`PyLongObject`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:272
msgid ""
"If the value of *obj* is out of range for an :c:type:`unsigned long long`, "
"return the reduction of that value modulo ``PY_ULLONG_MAX + 1``."
msgstr ""

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:275
msgid ""
"Returns ``(unsigned long long)-1`` on error.  Use :c:func:`PyErr_Occurred` "
"to disambiguate."
msgstr ""
"出错时返回 ``(unsigned long long)-1``，请利用 :c:func:`PyErr_Occurred` 辨别具"
"体问题。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:281
msgid ""
"Return a C :c:type:`double` representation of *pylong*.  *pylong* must be an "
"instance of :c:type:`PyLongObject`."
msgstr ""
"返回 *pylong* 的 C 语言 :c:type:`double` 形式。*pylong* 必须是 :c:type:"
"`PyLongObject` 的实例。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:284
msgid ""
"Raise :exc:`OverflowError` if the value of *pylong* is out of range for a :c:"
"type:`double`."
msgstr ""
"如果 *pylong* 的值超出了 :c:type:`double` 的取值范围则会引发 :exc:"
"`OverflowError`。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:287
msgid ""
"Returns ``-1.0`` on error.  Use :c:func:`PyErr_Occurred` to disambiguate."
msgstr "出错时返回  ``-1.0`` ，请利用 :c:func:`PyErr_Occurred` 辨别具体问题。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:292
msgid ""
"Convert a Python integer *pylong* to a C :c:type:`void` pointer. If *pylong* "
"cannot be converted, an :exc:`OverflowError` will be raised.  This is only "
"assured to produce a usable :c:type:`void` pointer for values created with :"
"c:func:`PyLong_FromVoidPtr`."
msgstr ""
"将一个 Python 整数 *pylong* 转换为 C 语言的 :c:type:`void` 指针。 如果 "
"*pylong* 无法转换，则会触发 :exc:`OverflowError`。 这只是保证为 :c:func:"
"`PyLong_FromVoidPtr` 创建的值产生一个合法的  :c:type:`void` 指针。"

#: /home/runner/work/python-docs-tx-translations/python-docs-tx-translations/cpython/Doc/c-api/long.rst:297
msgid ""
"Returns ``NULL`` on error.  Use :c:func:`PyErr_Occurred` to disambiguate."
msgstr "出错时返回  ``NULL``，请利用 :c:func:`PyErr_Occurred` 辨别具体问题。"
